chat-plugins/namefilter.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+const DIR = 'config/namefilter.json';
+let namefilter;
+try {
+	namefilter = JSON.parse(fs.readFileSync(DIR));
+} catch (err) {
+	fs.writeFileSync(DIR, '{}');
+	namefilter = JSON.parse(fs.readFileSync(DIR));
+}
+
+let notifyName = exports.notifyName = function (user) {
+	for (let i in namefilter) {
+		if (i == user.userid || user.userid.indexOf(i) > -1) {
+			Rooms('staff').add("|c|~| [NameFilter] User " + user + "'s name contains the filtered word '" + i + '"').update();
```

Comment:
Let's make this just a regular message. As in, make this line something like:

``Rooms('staff').add("[Name Filter] Should probably FR: <b>" + user + "</b>").update();``